app/controllers/course/experience_points_disbursement_controller.rb

```diff
@@ -22,7 +18,17 @@ def create # :nodoc:
 
   private
 
-  def experience_points_disbursement_params # :nodoc:
+  def load_and_authorize_experience_points_disbursement # :nodoc:
+    @experience_points_disbursement ||=
+      Course::ExperiencePointsDisbursement.new(current_course, send("#{params[:action]}_params"))
```

Comment:
Use `action_name` instead of `params` from user input. 

Is there a `course_id` param ? we can assign course id in the similar way as group_id.